pkg/machinepool/machinepool.go

```diff
@@ -2118,6 +2142,23 @@ func getNodePoolReplicas(cmd *cobra.Command,
 		IsHostedCp:     true,
 	}
 
+	// Get capacity reservation limits for interactive mode
+	var capacityReservationId string
+	var availableInstances int32
+	if interactive.Enabled() && nodePool != nil && nodePool.AWSNodePool() != nil &&
+		nodePool.AWSNodePool().CapacityReservation() != nil {
+		capacityReservationId, _ = nodePool.AWSNodePool().CapacityReservation().GetId()
+		if capacityReservationId != "" && r != nil && r.AWSClient != nil {
+			_, available, queryErr := r.AWSClient.GetCapacityReservationDetails(capacityReservationId)
+			if queryErr == nil {
+				availableInstances = available
+				// Show capacity reservation info to user
+				r.Reporter.Infof("Capacity reservation '%s' has %d available instance(s)",
+					capacityReservationId, availableInstances)
+			}
```

Comment:
Why are we only handling the positive flow? I would think users should know what the error is from AWS

Reply:
@olucasfreitas ^